sunpy/coordinates/wcs_utils.py

```diff
@@ -36,7 +41,7 @@ def solar_wcs_frame_mapping(wcs):
     ycoord = wcs.wcs.ctype[1][0:4]
 
     if xcoord == 'HPLN' and ycoord == 'HPLT':
-        return Helioprojective(dateobs=dateobs, observer=observer)
+        return Helioprojective(dateobs=dateobs, observer=observer, rsun=rsun)
```

Comment:
Yes, Helioprojective already had a rsun attribute it just wasn't getting auto filled by Map.